app/code/community/Ebizmarts/MailChimp/Helper/Data.php

```diff
@@ -4290,4 +4290,34 @@ public function getApiKeyValue()
         $scopeArray = $this->getCurrentScope();
         return $this->getApiKey($scopeArray['scope_id'], $scopeArray['scope']);
     }
+
+    /**
+     * @return Mage_Sales_Model_Order
+     */
+    public function getSessionLastRealOrder()
+    {
+        $checkoutSession = $this->getCheckOutSession();
+        $order = $checkoutSession->getLastRealOrder();
+        if(is_null($order)){
```

Comment:
change `if(is_null($order)){` to `if (is_null($order)) {` for better readability